assets/css/components/LessonList.scss

```diff
@@ -40,10 +40,11 @@
       @include title-2;
       @include make-col-ready();
       width: auto;
+      line-height: 3.2rem;
```

Comment:
The issue I was trying to solve with this (and the following) change was the one you can see in the attached images.

This is how it is with the previous heights:
![course titles master](https://user-images.githubusercontent.com/5033684/67787181-ccc51b80-fa78-11e9-94c1-0545df2b60ac.png)

This is how it is with the heights I set (the titles are not uppercase because we removed the `upper`):
![course titles rad_lessons](https://user-images.githubusercontent.com/5033684/67787202-d51d5680-fa78-11e9-850e-f115c5cd5f07.png)

Reply:
I understand that the styling is bad. Nevertheless, please nix the change.

(Our stylesheets are a bug bonanza. Workarounds make it hard to fix our bugs. We have a ton of work to do in our CSS, and I hope we can deploy without doing any CSS work at all :).)

Reply:
OK. After all, with the changed URLs, it will be a while until we have to show a second row.